docs/content/access-api-rate-limits.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: Flow Access API Rate limits on DapperLabs Access nodes
+---
+
+#### Rate limits
+
+Following are the current rate limits for the [Access Node gRPC API](/access-api).
+Once the limit has reached, the client will receive a rpc error 1ResourceExhausted` in the gRPC response.
+
+Pleqse note, these limits only apply to the access nodes hosted by DapperLabs. Access node run by other node operators will have different rate limits.
+
+
+##### Mainnet
+
+|                 gRPC API                             | Total request per second per IP               |
+|:-----------------------------------------------------|:---------------------------------------------:|
+| flow.access.AccessAPI/Ping                           |                   2000                        |
```

Comment:
we have 4 public facing ANs with the limit set to 500 on each.

Reply:
It's probably worth adding a note about that above the table so people don't get confused if they send requests to an individual node.

The limits are the same for gRPC-Web as well right?

Reply:
added note abt both those things.

Reply:
any idea if they also use the same counters